iothub/device/src/ConnectionStatusInfo.cs

```diff
@@ -27,34 +27,23 @@ internal ConnectionStatusInfo(ConnectionStatus status, ConnectionStatusChangeRea
         /// <summary>
         /// The current connection status of the device or module.
         /// </summary>
-        /// <remark>
```

Comment:
What it defaults to is of no concern to a user.

Reply:
Is this combination (Disconnected, Client_Close) the first status, reason that is returned by this class? If yes, then it might be worth calling that out somewhere so that users know to expect it as the first reported status.

Reply:
Fair enough. I see your point.